### PR TITLE
fixes #1858: added a save command

### LINKED ISSUE #1858: Update the docs to add a save command
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/guides/getting-started/building-your-first-project/build-your-first-models#change-the-way-your-model-is-materialized

### What part(s) of the page would you like to see updated?

After updating the `models` config block, we need to click on the `save` button to save the changes to the file. Then only we can execute the `dbt run` command. Though it is generally understood that the changes need to be saved, it will be good if we can have an additional step to mention it explicitly in the documentation.

This is the current steps in the docs:
image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by itsadityagupta**
## Description & motivation
Fixes #1858: Added an additional step to `save` the file in the documentation.

## Prerelease docs
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

### CODE REVIEW

**Comment by itsadityagupta** [comment]
I'm not sure whether I've to update Versioning docs for these changes. If needed, please let me know.

**Review by runleonarun — APPROVED** [review]
Thank you, @Aditya-Gupta1, for your contribution!

File: website/snippets/tutorial-change-way-model-materialized.md [hunk 1]
```diff
@@ -24,13 +24,14 @@ By default, everything gets created as a view. You can override that by material
...
       </File>
+    - Click **save**.
 
 2. Enter the `dbt run` command. Your `customers` model should now be built as a table!
     :::info
     To do this, dbt had to first run a `drop view` statement (or API call on BigQuery), then a `create table as` statement.
     :::
 
-3. Edit `models/customers.sql`  to override the `dbt_project.yml` for the `customers` model only by adding the following snippet to the top:  
+3. Edit `models/customers.sql`  to override the `dbt_project.yml` for the `customers` model only by adding the following snippet to the top, and click **save**:
```
Comment (runleonarun) [inline website/snippets/tutorial-change-way-model-materialized.md]:
```suggestion
3. Edit `models/customers.sql`  to override the `dbt_project.yml` for the `customers` model only by adding the following snippet to the top, and click **Save**:  
```

File: website/snippets/tutorial-change-way-model-materialized.md [hunk 2]
```diff
@@ -24,13 +24,14 @@ By default, everything gets created as a view. You can override that by material
       ```
 
       </File>
+    - Click **save**.
```
Comment (runleonarun) [inline website/snippets/tutorial-change-way-model-materialized.md]:
```suggestion
    - Click **Save**.
```

**Review by runleonarun — COMMENTED** [review]
Looks good! We can capitalize "Save" here

**Pull request merged**